src/index.html

```diff
@@ -8,6 +8,7 @@
 
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="bundle.css">
```

Comment:
This is included in the instructions but should not be part of the default HTML file since it won't exist.